coalib/bearlib/abstractions/ExternalBearWrap.py

```diff
@@ -4,7 +4,7 @@
 from collections import OrderedDict
 
 from coalib.bears.LocalBear import LocalBear
-from coala_decorators.decorators import enforce_signature
+from coala-utils.decorators import enforce_signature
```

Comment:
HOLD ON - there's a minus in the package name? That leads to problems. Please rename and make a bugfix release!